db/migrate/20160907095103_add_availability_to_listing_shape.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailabilityToListingShape < ActiveRecord::Migration
+  def change
+    add_column :listing_shapes, :availability, :string, limit: 32, default: "none", after: :shipping_enabled
```

Comment:
This column accepts only a predefined set of strings, is it possible to make it an enumeration?